test/verify/machineslib.py

```diff
@@ -209,6 +209,7 @@ def startVm(self, name, graphics='spice', ptyconsole=False):
         img = "/var/lib/libvirt/images/{0}-2.img".format(name)
         m.upload([image_file], img)
         m.execute("chmod 777 {0}".format(img))
+        self.addCleanup(self.machine.execute, "rm -f {0}".format(img))
```

Comment:
Why was the `-f` added? Do some tests remove the file by itself?

Reply:
Yes, at least one or two tests remove this file.